test/Concurrency/predates_concurrency.swift

```diff
@@ -249,3 +249,20 @@ extension MainActorPreconcurrency: NotIsolated {
     }
   }
 }
+
+open class Parent {
+    init(userInfo: [String : Any]) {
+        self.userInfo = userInfo
+    }
+
+  @preconcurrency open var userInfo : [String : any Sendable] // expected-note {{overridden declaration is here}}
+}
+
+class Child : Parent {
+    override var userInfo: [String : Any] { // expected-warning {{declaration 'userInfo' has a type with different sendability from any potential overrides; this is an error in the Swift 6 language mode}}
```

Comment:
I think it would be good to add a few more test cases here. For example one with global actor attributes i.e. `@escaping @MainActor (any Sendable) -> Void` to make sure all that works as expected. 

And a few where child properties adopt `any Sendable` to make sure that no warnings are emitted.